.travis.yml

```diff
@@ -11,5 +12,4 @@ script:
   - mypy --ignore-missing-imports .
   - pytest . --doctest-modules
 after_success:
-  - scripts/build_directory_md.py > DIRECTORY.md
-  - cat DIRECTORY.md
+  - scripts/build_directory_md.py 2>&1 | tee DIRECTORY.md
```

Comment:
Why this change?  How is it different than the current code?

Reply:
This change eliminates a line and does both operations. Printing the output, and writing it to the file, see [tee](https://en.wikipedia.org/wiki/Tee_(command))